Source/WebCore/rendering/RenderFragmentContainer.cpp

```diff
@@ -319,26 +319,26 @@ void RenderFragmentContainer::detachFragment()
 RenderBoxFragmentInfo* RenderFragmentContainer::renderBoxFragmentInfo(const RenderBox* box) const
```

Comment:
Should the `box` parameter be a Ref, too, since we assume it is never `nullptr`, perhaps as follow-up work?

Reply:
We normally don't switch parameter types to smart pointers like this unless there is ownership transfer, to avoid refcounting churn. However, you're right that it is assumed to be non-null so it should probably be a C++ reference. I'll check how big of a change that is, as I suspect this is a virtual function.